### PR TITLE
[AIRFLOW-3933] Fix various typos

### PR DESCRIPTION
**Pull request by ryanyuan**
Fix typos

Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow-3933](https://issues.apache.org/jira/browse/AIRFLOW-3933) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by zhongjiajie** [comment]
@ryanyuan Awesome!

File: airflow/contrib/operators/dataproc_operator.py [hunk 1]
```diff
@@ -1376,7 +1376,7 @@ def execute(self, context):
         self.hook.wait(self.start())
 
     def start(self, context):
-        raise AirflowException('plese start a workflow operation')
+        raise AirflowException('please start a workflow operation')
```
Comment (drewsonne) [inline airflow/contrib/operators/dataproc_operator.py]: Not sure what the convention is, but maybe should be capitalised?

File: dev/airflow-pr [hunk 2]
```diff
@@ -63,7 +63,7 @@ AIRFLOW_GIT_LOCATION = os.environ.get(
     "AIRFLOW_GIT",
     os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
 
-# Remote name which points to the Gihub site
+# Remote name which points to the Github site
```
Comment (drewsonne) [inline dev/airflow-pr]: I think it should be `GitHub`

**Comment by XD-DENG** [comment]
@ryanyuan awesome!

A minor minor suggestion: if all changes in your PR are purely doc change, you can add "[ci skip]" in your git commit message to skip the CI, so that it can save ASF a small amount of money.

**Comment by zhongjiajie** [comment]
@XD-DENG I donnot know CI test need money before. Isn't travis-ci free for open source project?

**Comment by XD-DENG** [comment]
Hi @zhongjiajie , Yes it is free for open source projects, but for relatively small size ones. If you integrate your Airflow fork with TravisCI, you will find that it's free but you can only run 5 tests concurrently.

ASF is using a subscribed service from Travis so Apache projects like Airflow's CI on Travis can run "unlimited" concurrent tests. It's not free in that case.

I got to know this from Ash earlier: https://github.com/apache/airflow/pull/3967#issuecomment-425369893 (both "[ci skip]" and "[skip ci]" would work I remember)

**Comment by ryanyuan** [comment]
@XD-DENG Thanks for the tips!

**Comment by zhongjiajie** [comment]
@XD-DENG I get this, thanks.

**Comment by Fokko** [comment]
Thanks @ryanyuan

**Pull request merged**